test/views/components/UnorderedListSpec.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.components
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import utils.SpecBase
+import views.html.components.unorderedList
+
+class UnorderedListSpec extends SpecBase {
+
+  "component" should {
+
+    "not display the unordered list when li msg list is empty" in {
+      val id = "testId"
```

Comment:
val id = "testId" could be removed from every test and add to a Setup trait

Reply:
Its not good to add a Setup for only one val as Setup trait due to performance reasons. I have moved the id into TestData and being used from there.